Description: Make `serde` feature depend on `std`.
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Forwarded: https://github.com/garro95/priority-queue/pull/78
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,6 +47,7 @@ rustdoc-args = [
 benchmarks = []
 default = ["std"]
 std = ["indexmap/std"]
+serde = ["std", "dep:serde"]
 
 [lib]
 name = "priority_queue"
